manifests/g/Giorgiotani/Peazip/8.4.0/GiorgioTani.PeaZip.installer.yaml

```
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: GiorgioTani.PeaZip
+PackageVersion: 8.4.0
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- 7z
+- ace
+- arc
+- bz2
+- cab
+- gz
+- iso
+- paq
+- pea
+- rar
+- tar
+- wim
+- zip
+- zipx
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WIN64.exe
+  InstallerSha256: A1BD79AA0FA46B2041CE807202E9BBA9F1BBB2DC3EFDA76C1DB93C146DFC83FD
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WIN64.msi
+  InstallerSha256: 6E3818E72E3751DD919A5366B5D191896D8EEF8B644A002C7644DADD00BCDDBC
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WINDOWS.exe
+  InstallerSha256: 03D7B50F22D815468E47E6B58ACF008C94247E2E781232E26E4B466149DB5583
```

These are again flipped around
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WIN64.exe
  InstallerSha256: A1BD79AA0FA46B2041CE807202E9BBA9F1BBB2DC3EFDA76C1DB93C146DFC83FD
- Architecture: x64
  InstallerType: msi
  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WIN64.msi
  InstallerSha256: 6E3818E72E3751DD919A5366B5D191896D8EEF8B644A002C7644DADD00BCDDBC
- Architecture: x86
  InstallerType: exe
  InstallerUrl: https://github.com/peazip/PeaZip/releases/download/8.4.0/peazip-8.4.0.WINDOWS.exe
  InstallerSha256: 03D7B50F22D815468E47E6B58ACF008C94247E2E781232E26E4B466149DB5583
```

---

YamlCreate's InputObject would fix it since then I would be able to _assign_ URLs to Architecture.